src/VtkOutputter.cpp

```diff
@@ -130,11 +134,11 @@ namespace blitzdg {
 
 	void VtkOutputter::writeFieldsToFiles(std::map<std::string, real_matrix_type>& fields, index_type tstep) {
 		for (auto kv : fields) {
```

Comment:
This loop creates a copy in `kv` of each element in `fields`. You may want to write it as
`for(const auto& kv : fields)`